### ISSUE TITLE
_iso-Feature Installation not possible (only tested with UEFI environment)

### ISSUE THREAD

**Issue by dnan0s** [gardenlinux/gardenlinux#1830]
The `_iso`-Feature is meant to boot as liveboot and be installed on another disk with `/opt/install/install.sh`, if i understand correctly. This script seems to not be current.

**What happened:**
1. When running the script, it fails first, because it does not find files like `install.fstab` and `install.part`. The script only works, if the current directory is the path `/opt/install/ - `I suppose placing ${thisdir} in front of it all occurrences makes it work from all paths. 
2. If running it after fixing step 1, the script cries because of missing file ` /etc/kernel/postinst.d/zz-kernel-install`. My enviroment was UEFI, i did not test it in BIOS, yet. If i do, i'll update this issue. I do not specifically know what `zz-kernel-install` is doing, but i just thought it's not included anymore, so i can comment it out, which i did. Then the script ran through, but it does not produce a bootable Image. 
I think, i am missing some understanding regarding the `zz-kernel-install` file to fix this issue myself. 

**Debug Logs:**
Not at hand, might update in the future!

**How to reproduce it**:
- Basically run `/opt/install/install.sh`

PS: No time pressure on our side, i've just experienced his - we chose another way now.

**Comment by Vincinator** [comment]
Hi @dnan0s thanks for reporting. 
I assume you are using latest Garden Linux main branch? Or is this related to the rel-934?

**Comment by dnan0s** [comment]
Yeah, i've been building on the latest Branch :+1:
